### ISSUE TITLE
deleting a catalog won't remove albums, artists, etc.

### ISSUE THREAD

**Issue by krodelabestiole** [ampache/ampache#1013]
I use Ampache 3.8.0 on debian and ran into few issue. I've used the standard web gui installation and everything went fine apparently.
One thing i'm not sure about is the "secret key" ? i guess it has not been changed during the installation... Should i have changed it manually ? At what time ?
Now i've got a bigger problem : when i delete a catalog it won't remove any of the albums, artists, stats, etc. it contains. Here is the log (i changed the keys) :
> 2015-11-19 20:43:05 [krodelabestiole] (session) -> 1234567890abcdef has been extended to Thu, 19 Nov 2015 21:43:05 +0100 extension length 3600 
2015-11-19 20:43:05 [krodelabestiole] (session) -> Session created: f5583aaf188b2e6e8e1922d1a10f2f6a 
2015-11-19 20:43:05 [krodelabestiole] (Core) -> Registered post form delete_catalog with SID 56bd936fd49d67d58b138dcefd7f931e and expiration 1447965785 (3600 seconds from now) 
2015-11-19 20:43:07 [krodelabestiole] (session) -> 1234567890abcdef has been extended to Thu, 19 Nov 2015 21:43:07 +0100 extension length 3600 
2015-11-19 20:43:07 [krodelabestiole] (session) -> Session created: fedcba9876543210 
2015-11-19 20:43:07 [krodelabestiole] (Core) -> Verified SID 56bd936fd49d67d58b138dcefd7f931e for post form delete_catalog 
2015-11-19 20:43:08 [krodelabestiole] (catalog) -> Database cleanup started 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:09 [krodelabestiole] (Dba) -> Error: ["42S02",1146,"Table 'ampache.label' doesn't exist"] 
2015-11-19 20:43:10 [krodelabestiole] (catalog) -> Database cleanup ended 
2015-11-19 20:43:10 [krodelabestiole] (Core) -> Registered post form confirmation with SID 13d25b4b5c7a6398878b6ecabd9c0280 and expiration 1447965790 (3600 seconds from now) 
2015-11-19 20:43:15 [krodelabestiole] (session) -> 1234567890abcdef has been extended to Thu, 19 Nov 2015 21:43:15 +0100 extension length 3600 
2015-11-19 20:43:15 [krodelabestiole] (ajax.server.php) -> Called for page: {index} 

well it's true that the database doesn't contain any "label" table. But i haven't changed anything manually...
update_info.db_version says 370035 if that's of any help

**Comment by krodelabestiole** [comment]
updated to 3.8.1
i don't have this issue anymore

**Comment by Afterster** [comment]
Yes that's what I thought :). Thanks for the feedback.
